packages/app/src/specs/InlineSpecList.vue

```diff
@@ -1,9 +1,16 @@
 <template>
   <div class="w-280px">
+    <CreateSpecModal
```

Comment:
We have this twice already, once on `NoSpecPage.vue` and once on `SpecsList.vue`. I wonder if we need it a third time (or, do we even need it twice?)

What do you think about hoisting all the modals up to the top level, and moving the `showModal` state from a `ref` to a `modalStore` or something like that? Then we can just have one instance of the modal rendered, and a central place to manage all the modals.

Reply:
I think we would run into complications trying to hoist the modal up, and even then I don't think it's a good idea. We only have one modal right now, but if/when they grow, I would find it strange to have a bunch of un-rendered modals sitting at the top. The component declaration might be duplicated (and the state for handling the open/close state), but the state for managing its state should live within the context that the modal was opened so I'm ok with it.

Currently, each page runs it's own query, so to properly lift the modal up we would have to change that structure since the modal relies on gql data.

Reply:
Ok, let's do this for now, but keep this is mind moving forward 👍